conan_provider.cmake

```diff
@@ -239,6 +239,10 @@ function(detect_compiler COMPILER COMPILER_VERSION COMPILER_RUNTIME COMPILER_RUN
         set(_COMPILER "clang")
         string(REPLACE "." ";" VERSION_LIST ${CMAKE_CXX_COMPILER_VERSION})
         list(GET VERSION_LIST 0 _COMPILER_VERSION)
+    elseif(_COMPILER MATCHES TIClang)
+        set(_COMPILER "ticlang")
```

Comment:
This would be an issue. As the ``ticlang`` is not a built-in ``settings.yml``, it will fail for Conan users. Maybe it should be added first there? Or why not just assimilating it to ``clang``? Is it binary incompatible to ``clang``? Different versioning? A bit of scope and extra info about this compiler would be nice